scripts/game/GameScores.js

```diff
@@ -1,19 +1,33 @@
+import{useTeams} from "../teams/TeamProvider.js"
+
 const eventHub = document.querySelector(".container")
 const contentTarget = document.querySelector(".main--game-play")
 
-const render = (teamObj) => {
-    return `
+eventHub.addEventListener("allTeamsSelected", (customEvent) => {
+   const team1 = parseInt(customEvent.detail.team1Id)
+   const team2 = parseInt(customEvent.detail.team2Id)
+   const team3 = parseInt(customEvent.detail.team3Id)
```

Comment:
I'd like to make these variable names more explicit by calling them team1Id, team2Id, team3Id.

Reply:
I second Gib's change